src/modules/Lan.rb

```diff
@@ -780,8 +780,12 @@ def read_config(report: true)
     #   parameter is given then all changes will be written
     #
     # @see Y2Network::ConfigWriter
-    def write_config(only: nil)
-      yast_config.write(original: system_config, only: only)
+    def write_config(only: nil, report: true)
+      result = yast_config.write(original: system_config, only: only)
+      if result&.issues? && report
+        return false unless Y2Issues.report(result.issues) == :yes
```

Comment:
Beware that this API changed recently and the report method now returns a boolean. See https://github.com/yast/yast-network/pull/1240/files#diff-c536d97bfa2e800b076011ec2976dae8b9c77b83fec29c7a1741554d1f8750c0.